gpu/containers/src/initialization.cpp

```diff
@@ -39,6 +39,8 @@
 
 #include "cuda.h"
 #include <cstdio>
+#include <array>
+#include <algorithm>
```

Comment:
Why is this needed? Please add `// for ...` in the added headers

Reply:
Oh, bother! Sorry - leaving the algo header was a mistake. I replaced the raw loop with a `std::find_if` upon the advice of my linter. I tried it but didn't want to push the result and left the include by mistake. If you agree with the linter's sentiment, I can do that again. Does PCL have any code policy about that?

Reply:
Not really. In that situation, a return is easier to read.